src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int raduis) {
+        super(color);
+        this.radius = raduis;
+    }
+
+    public Circle() {
```

Comment:
Don't need to create methods (setRadius) or constructors that you actually don't use. 

Check other figures